applications/fishing-map/src/features/map/map.hooks.ts

```diff
@@ -192,7 +187,7 @@ export const useClickedEventConnect = () => {
         return false
       }
       const isFeatureVisible = feature.temporalgrid.visible
-      const isFishingFeature = feature.temporalgrid.sublayerId.startsWith('viirs')
+      const isFishingFeature = feature.temporalgrid.sublayerInteractionType === 'viirs'
       return isFeatureVisible && isFishingFeature
```

Comment:
copied and pasted from above... and I forgot to rename to `isViirsFeature` just fixed